source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -198,6 +198,8 @@ You must:
 - [create the new PaaS-hosted PostgreSQL database](/deploying_services/postgresql/#set-up-a-postgresql-service)
 - [target the space](/deploying_apps.html#set-a-target) where your new database is located
 
+#### Please note that the import operations described in this section are expected to emit error messages. The errors are detailed at the end of this section.
+
```

Comment:
I would add this under step 2 of each of the sections to make it clear where it matters, and link to the section where the errors are detailed
```suggestion
    You will see some error messages when importing the data file into the target database. [These errors are expected](/deploying_services/postgresql/#import-data-expected-errors) and the import process will succeed.'


```

Reply:
oh I see. I added the wording here, but my suggestion was to put this sentence under step 2 of each of the sections, so you would need to manually change it